lib/propolis/src/block/crucible.rs

```diff
@@ -37,16 +37,24 @@ impl CrucibleBackend {
         request: VolumeConstructionRequest,
         read_only: bool,
         producer_registry: Option<ProducerRegistry>,
+        log: slog::Logger,
     ) -> Result<Arc<Self>> {
-        CrucibleBackend::_create(gen, request, read_only, producer_registry)
-            .map_err(map_crucible_error_to_io)
+        CrucibleBackend::_create(
+            gen,
+            request,
+            read_only,
+            producer_registry,
+            log,
+        )
+        .map_err(map_crucible_error_to_io)
     }
 
     fn _create(
         gen: u64,
         request: VolumeConstructionRequest,
         read_only: bool,
         producer_registry: Option<ProducerRegistry>,
+        log: slog::Logger,
```

Comment:
Ohh, nice.  Maybe I (we, we being Crucible) can pass this log down to crucible and have it send message there instead of stdout like it does now.